.github/workflows/release_pull_request.yml

```diff
@@ -1,4 +1,4 @@
-name: pull request
+name: release pull request
 
 on:
   pull_request:
```

Comment:
similar to the other comment, this can be updated to `on: push: branches: [release-please-*]` to not create any jobs. not critical since I'm changing it in #2300 anyways